speech/cloud-client/README.md

```diff
@@ -155,7 +155,11 @@ mvn exec:java -DRecognize -Dexec.args="word-level-conf gs://cloud-samples-tests/
 ```
 
 ## Infinite Streaming
-Continuously stream audio to the speech API over multiple requests 
+Continuously stream audio to the speech API over multiple requests (by default en-US).
 ```
 mvn exec:java -DInfiniteStreamRecognize
 ```
+If stream audio is in different language, you could also pass language code as a command line argument (for example, korea).
+```
+mvn exec:java -Dexec.args="-lang_code=ko-KR" -DInfiniteStreamRecognize
+```
```

Comment:
(Based on your idea) do you want to also link to the full list of language codes?

https://cloud.google.com/speech-to-text/docs/languages

Reply:
Or at least english?